crates/viewer/re_view_spatial/src/eye.rs

```diff
@@ -574,3 +571,31 @@ impl ViewEye {
         self.center += translate;
     }
 }
+
+// Logic should be similar to `impl TypedComponentFallbackProvider<LinearSpeed> for SpatialView3D`
+impl TypedComponentFallbackProvider<LinearSpeed> for ViewEye {
+    fn fallback_for(&self, ctx: &re_viewer_context::QueryContext<'_>) -> LinearSpeed {
+        let maybe_state = re_viewer_context::ViewStateExt::downcast_ref::<crate::SpatialViewState>(
+            ctx.view_ctx.view_state,
+        );
+        let speed = match maybe_state {
+            Ok(spatial_view_state) => {
+                let bounding_boxes = &spatial_view_state.bounding_boxes;
+                self.fallback_speed_for_mode(bounding_boxes) as f64
+            }
+            Err(view_system_execution_error) => {
+                re_log::error!("Error while downcasting {}", view_system_execution_error);
+                1.0_f64
+            }
+        };
+        LinearSpeed(re_types::datatypes::Float64(speed))
+    }
+}
+
+impl TypedComponentFallbackProvider<Eye3DKind> for ViewEye {
+    fn fallback_for(&self, _ctx: &re_viewer_context::QueryContext<'_>) -> Eye3DKind {
+        Eye3DKind::default()
+    }
+}
```

Comment:
just fyi: I think it's fine to keep this here but note that this is technically not needed since iff a default is available on a given component type, it will automatically pick that if there's no fallback provider
(very non-obvious, I know. Need to document this mechanic better)